vendor/assets/schemas/lom_1484.12.3-2020/lom.xsd

```diff
@@ -0,0 +1,102 @@
+         composite schema for validation of the following:          
+
+         1) The use of LOM Base Schema vocabulary source/value pairs only
+         2) Uniqueness constraints defined by 1484.12.1-2002
+         3) No existenace of any defined extensions
+
+         Alternative composite schemas can be assembled by selecting
+         from the various alternative component schema listed below.
+      </xs:documentation>
+      <xs:documentation>
+        *****************************************************************************
+        **                           CHANGE HISTORY                                **
+        *****************************************************************************
+        ** 11/14/2003:  1)Updated comment describing vocab/strict.xsd.  Indicated  **
+        **                that the strict.xsd is used to validate vocabularies     **
+        **                defined in the LOM V1.0 Base Schema.                     **
+        **              2)Moved included schema elementNames.xsd just before       **
+        **                elementTypes.xsd.                                        **
+        **              3)Moved the element declaration for the top-level lom      **
+        **                metadata element to a separate file (rootElement.xsd)    **
+        **                and included this file just after elementTypes.xsd.      **
+        **              4)Moved the XML Schema import statements before the XML    **
+        **                Schema include statements.                               **
+        **              5)Moved the element group declaration named                **
+        **                lom:customElements to a separate file (anyElement.xsd)   **
+        **                and included this new file just before the XML Schema    **
+        **                import statments.                                        **
+        **                                                                         **
+        ** 03/15/2004:  1)Switched which vocabulary vaidation approach is the      **
+        **                default to be used by this template from vocab/loose.xsd **
+        **                to vocab/strict.xsd.  Base on ballot resolution comment  **
+        **                addressed on 03/15/2004                                  **
+        **              2)Switched the extension validation approach to use        **
+        **                extend/strict.xsd.  The XSD now is reprsentative of      **
+        **                a schema that can be used to validate strictly           **
+        **                conforming LOM XML Instances.                            **
+        *****************************************************************************
+      </xs:documentation>
+   </xs:annotation>
+
+   <!-- Learning Object Metadata -->
+
+   <xs:include schemaLocation="vendor/assets/schemas/lom_1484.12.3-2020/common/anyElement.xsd"/>

```

Comment:
I had to adjust these paths in this file. I hope I can revert that when we move this to the test suite and then leave the file untouched.

Reply:
Fine for me. We're creating a prototype anyway.